processes/fulfillURLManifest.py

```diff
@@ -148,6 +154,15 @@ def fulfillFlagUpdate(self, metadata):
                                 newLinkFlag['fulfill_limited_access'] = True
                                 link.flags = newLinkFlag
                                 self.commitChanges()
+                        
+    def checkMetaData(self, checkRightsStatus, metadataJSON):
```

Comment:
checkMetadata is not a clear function name. What are we checking? 

we are trying to see if the fulfill_limited_acess flag is in the link right? Can we simplify this logic?

Reply:
I changed the name to check_rights_status which is what we are checking. The logic is as simplified as it can be given my understanding.